bluebird-promisell.js

```diff
@@ -0,0 +1,49 @@
+/* jshint esnext:true */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const Promise = require('bluebird');
+const P = require('bluebird-promisell');
+const R = require('ramda');
+const S = require('sanctuary');
+
+//    readFile :: Object -> String -> Promise Error String
+const readFile = S.flip(Promise.promisify(fs.readFile));
+
+//    readFileWithDirAndName :: String -> String -> Promise Error String
+const readFileWithDirAndName = R.curryN(2, (dir, name) => {
+  const filePath = path.join(dir, name);
+  return readFile({ encoding: 'utf8' })(filePath);
+});
```

Comment:
It's inconsistent for `readFileWithDirAndName` to specify the encoding while `readFile` requires the caller to provide it.